internal/testutils/tests.go

```diff
@@ -15,3 +17,17 @@ func SkipUnlessRoot(t *testing.T) {
 		t.Skip("Test has to be run as root, skipping...")
 	}
 }
+
+// WantError asserts the error value depending if an error is wanted and stops the test if requested.
+func WantError(t *testing.T, err error, wantErr, stopOnError bool) {
+	t.Helper()
+
+	if wantErr {
+		require.Error(t, err, "Expected %s to return an error, but it didn't", t.Name())
+		if stopOnError {
+			t.SkipNow()
```

Comment:
Yes, I agree. I checked the testing package to see if there's a way to mark the test as completed and passed, instead of skipped, but it seems the only way to do that is by finishing the test function. Maybe calling `os.Exit(0)` does it (I'm not sure how it would interact with parallel testing, though)?